P3120 Козаева Яна Александровна task-6

yanakozaeva opened the pull request:
modified:   src/main/cpp/t01_max.cpp
	modified:   src/main/cpp/t02_star.cpp
	modified:   src/main/cpp/t03_diag.cpp
	modified:   src/main/cpp/t04_swap.cpp
	modified:   src/main/cpp/t05_kdiag.cpp
	modified:   src/main/cpp/t06_cinema.cpp
	modified:   src/main/cpp/t07_snake.cpp
	modified:   src/main/cpp/t08_chess.cpp
	modified:   src/main/cpp/t09_spiral.cpp

yanakozaeva committed:
Changes to be committed:
	modified:   src/main/cpp/t01_max.cpp
	modified:   src/main/cpp/t02_star.cpp
	modified:   src/main/cpp/t03_diag.cpp
	modified:   src/main/cpp/t04_swap.cpp
	modified:   src/main/cpp/t05_kdiag.cpp
	modified:   src/main/cpp/t06_cinema.cpp
	modified:   src/main/cpp/t07_snake.cpp
	modified:   src/main/cpp/t08_chess.cpp
	modified:   src/main/cpp/t09_spiral.cpp

thejerome commented:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4951809&bid=16366581)

thejerome commented on src/main/cpp/t04_swap.cpp:
```diff
@@ -25,6 +25,42 @@
+	}
+	cin >> p >> q;
+	int count = 0;
+	int boof;
```
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'boof' can be reduced.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4951809)

thejerome commented on src/main/cpp/t04_swap.cpp:
```diff
@@ -25,6 +25,42 @@
+
+	}
+	cin >> p >> q;
+	int count = 0;
```
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'count' is assigned a value that is never used.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4951809)

thejerome commented on src/main/cpp/t07_snake.cpp:
```diff
@@ -17,11 +17,57 @@
+	int m;
+	cin >> n >> m;
+	int count = 0;
+	int numb = 0;
```
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'numb' is assigned a value that is never used.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4951809)

thejerome commented on src/main/cpp/t08_chess.cpp:
```diff
@@ -19,11 +19,65 @@
+	int m;
+	cin >> n >> m;
+	int count = 1;
+	int numb = 0;
```
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'numb' is assigned a value that is never used.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4951809)

yanakozaeva committed:
Changes to be committed:
	modified:   src/main/cpp/t04_swap.cpp
	modified:   src/main/cpp/t07_snake.cpp
	modified:   src/main/cpp/t08_chess.cpp

thejerome commented:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4951809&bid=16366581)

thejerome commented on src/main/cpp/t04_swap.cpp:
```diff
@@ -25,6 +25,42 @@
+	}
+	cin >> p >> q;
+	
+	int boof=0;
```
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'boof' is assigned a value that is never used.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4951809)

thejerome replied:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'boof' can be reduced.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4951809)

yanakozaeva committed:
Changes to be committed:
	modified:   src/main/cpp/t04_swap.cpp

yanakozaeva committed:
Changes to be committed:
	modified:   src/main/cpp/t04_swap.cpp

thejerome reviewed (commented):
Score: **70**
